Gillian-C/lib/parserAndCompiler.ml

```diff
@@ -107,9 +109,9 @@ module TargetLangOptions = struct
     Config.hide_mult_def := hide_mult_def
 end
 
-let compiled_progs = Hashtbl.create 16
+let compiled_progs = Hashtbl.create small_tbl_size
 
-let gil_paths = Hashtbl.create 16
+let gil_paths = Hashtbl.create small_tbl_size
```

Comment:
Do I have a way of resetting these tables ? 
I need it if I want to execute several programs without restarting the program